source/frontend/src/features/documents/ComposedDocument.ts

```diff
@@ -28,27 +30,40 @@ export class DocumentRow {
   isFileAvailable?: boolean;
   appCreateTimestamp?: string;
   appCreateUserid?: string;
+  relationshipId?: number;
+  relationshipType?: DocumentRelationshipType;
+  parentId?: number;
 
-  public static fromApi(document: Api_Document): DocumentRow {
+  public static fromApi(relationship: Api_DocumentRelationship): DocumentRow {
```

Comment:
just to keep in mind. Not sure if this is leaking abstraction given the row would only care about the document information vs also having the relationship information.